packages/@aws-cdk/aws-apigateway/README.md

```diff
@@ -153,6 +153,34 @@ plan.addApiStage({
 });
 ```
 
+The following example shows how to use a rate limited api key :
```

Comment:
Can you add a paragraph here on what RateLimitedApiKey class is and when someone would use this?

Reply:
Done!